lib/modules/apostrophe-pieces/public/js/manager-modal.js

```diff
@@ -619,6 +638,25 @@ apos.define('apostrophe-pieces-manager-modal', {
       self.hideSelectEverything();
       self.reflectChoiceCount();
     };
+
+    self.clearEverythingChoices = function() {
+      var $pieces = self.$el.find('[data-piece]');
+      
+      self.choices = [];
```

Comment:
Has this been tested with the chooser too? I think this needs to be self.clearChoices.

Reply:
DIdn't realize the chooser was at play here , will test.

One thing i'm coming across immediately playing with the chooser is that the chooser bar glitches off the page when scrolling and using select all .. different issue..

Reply:
To be "clear" I mean that `self.choices = []` needs to be a call to `self.clearChoices()` instead. The chooser overrides some of this stuff, if we stick to calling the methods that won't trip us up.